### DISCUSSION TITLE
When will V4 release?

### DISCUSSION THREAD

**Discussion by winner106** [remotion-dev/remotion#2424] [General]
When will V4 release?

When there are many components, webpack becomes slow. Is there any way to improve the bundling speed? For example, is it possible to use next.js 13 and remotion together to build a complete preview project in order to benefit from the speed boost provided by turbopack, if there are any such projects?

Thank you!

**Comment by JonnyBurger** [comment]
We aim to release Remotion 4.0 on July 3rd.

It's not possible to bundle a Remotion project using Turbopack - it is a project that integrates with Next.js but has no documentation with how to integrate with other frameworks. 

I looked at different bundlers like Vite and SWC, but came to the conclusion that no other bundler has all the features to make it suitable to replace Webpack - yet. This is also evident in that Webpack is still the default in Next.js.

The key for scaling is that not every render should create a new bundle. On the CLI, there is a cache, and for server-side rendering, a bundle should only be created initially and many renders should parametrize the same bundle.

If you are still suffering from slow bundles, then provide some info about your project architecture and we can take a look at it.

**Reply by winner106** [reply]
Thank you so much, I made a misunderstanding.
